### PR TITLE
Remove 005, fix 007 container path reference

### PR DESCRIPTION
**Pull request by jesse-peters**
- Fixes GHA concurrency issue with docker container builds
- Adds back readme images
- Some small naming issues on the GHA steps

### CODE REVIEW

**Comment by krisnova** [comment]
Please squash into a single commit once you are finished

**Comment by jesse-peters** [comment]
Thanks, that's a bad habit I'm still getting past. I'll keep working on it!

**Comment by jesse-peters** [comment]
Looking at why 036 wasnt run

**Comment by jesse-peters** [comment]
Alright we should be set. Whats best practice reagarding squashing in a case like this where there is a single commit after fixing a merge issue?

File: .github/workflows/007-cargo-udeps-aurae-builder-make-build-check-deps.yml [hunk 1]
```diff
@@ -41,9 +41,9 @@ env:
  CARGO_TERM_COLOR: always
 jobs:
  build-container:
-   uses: ./.github/workflows/031-ubuntu-aurae-builder-image-ubuntu-aurae-builder-docker-cargo-udeps-build.yml
- build-docs-with-container:
-   name: Build Docs
+   uses: ./.github/workflows/031-ubuntu-aurae-builder-image-ubuntu-docker-cargo-udeps-build.yml
+ check-udeps:
+   name: Check udeps
```
Comment (future-highway) [inline .github/workflows/007-cargo-udeps-aurae-builder-make-build-check-deps.yml:46]: *I can only comment on lines with a change*

Can you remove the build step (ln 69-70). cargo-udeps does its own build and it won't share the build from `make build` so we are building an extra time without benefit.

**Review by future-highway — CHANGES_REQUESTED** [review]
Blocking to indicate that this should not be merged until the PR check based on workflow 035 is set as required by an admin, replacing 005, which this PR removes.

**Comment by jesse-peters** [comment]
Thanks @future-highway

File: .github/workflows/007-cargo-udeps-aurae-builder-make-build-check-deps.yml (hunk 1 above)
Comment (jesse-peters) [reply .github/workflows/007-cargo-udeps-aurae-builder-make-build-check-deps.yml:46]: I can, but my concern with that is that cargo-udeps needs to install `libssl-dev`, and I didn't want to pollute the `make build` cache. I may have misunderstood the directions to not add those, so please correct me if I'm wrong 

https://github.com/aurae-runtime/aurae/blob/main/images/Dockerfile.build#L110

Comment (krisnova) [reply .github/workflows/007-cargo-udeps-aurae-builder-make-build-check-deps.yml:46]: Its fine for things like cargo-udeps I guess. I want to make sure we aren't using GNU libraries for OUR dependencies.

**Review by krisnova — APPROVED** [review]

File: .github/workflows/007-cargo-udeps-aurae-builder-make-build-check-deps.yml (hunk 1 above)
Comment (jesse-peters) [reply .github/workflows/007-cargo-udeps-aurae-builder-make-build-check-deps.yml:46]: It was a misunderstanding on my part, we resolved it in discord. We'll continue to split the docker image into stages so the build stage won't have the extra libraries. I was using the incorrect make command in the GHA job which resolved our issue.

**Pull request merged**